ipapython/ipautil.py

```diff
@@ -1647,3 +1647,14 @@ def rmtree(path):
             shutil.rmtree(path)
     except Exception as e:
         logger.error('Error removing %s: %s', path, str(e))
+
+
+def datetime_from_timestamp(t, units=1000):
```

Comment:
Could you please add a doc string that explains what ``t`` and ``units`` mean? The function name does not reflect that the function returns an UTC time stamp.

Reply:
it returns 'datetime.datetime' object, not a timestamp. I'll add docstring.